src/elements/Img-modal.vue

```
@@ -1,67 +1,55 @@
+  const props = defineProps<{ image: ImageField }>()
+
+  const thumbnailField = (props.image as ImageField<'thumbnail'>).thumbnail ?? props.image
+
+  const fullWidthImage = ref(false)
 </script>
 
+<template>
+  <button class="thumbnail-btn" @click="fullWidthImage = true">
+    <PrismicImage :field="thumbnailField" width="200" height="300" loading="lazy" />
+  </button>
+  <dialog :open="fullWidthImage" class="full" @click.self="fullWidthImage = false">
```

ESLint

[error] Replace `·:open="fullWidthImage"·class="full"·@click.self="fullWidthImage·=·false"` with `⏎····:open="fullWidthImage"⏎····class="full"⏎····@click.self="fullWidthImage·=·false"⏎··`

(prettier/prettier)